backend/backend/settings/common.py

```diff
@@ -12,16 +12,15 @@
 
 import json
 import os
-import pathlib
 import sys
 from datetime import timedelta
 
 import structlog
 from django.core.files.storage import FileSystemStorage
 
-from libs.gen_secret_key import write_secret_key
 from substrapp.compute_tasks.errors import CeleryRetryError
 
+from .deps.jwt import *
```

Comment:
why do we need a `import *`? 
It's usually considered bad practice :/ 
(same for the line below though)